### PR TITLE
[JENKINS-23447] related , avoid NPE if there are no getSensibleVariables()

### PR DESCRIPTION
**Pull request by dshvedchenko**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPluginAction.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public Object getTarget() {
         return new EnvInjectVarList(Maps.transformEntries(envMap,
                 new Maps.EntryTransformer<String, String, String>() {
                     public String transformEntry(String key, String value) {
-                        return getSensibleVariables().contains(key) ? "********" : value;
+                        return getSensibleVariables() != null && getSensibleVariables().contains(key) ? "********" : value;
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPluginAction.java]: In order to be 100%-sure about the value, call the method and assign its output to a variable before checking return values

**Comment by dshvedchenko** [comment]
@oleg-nenashev , is there any chances that plugin 1.92 will be released soon ?

**Comment by oleg-nenashev** [comment]
It makes sense to release a new version ASAP due to its possible impact.
I'll try to perform some additional testing tomorrow and then release the new version

**Comment by gboissinot** [comment]
Thank you for your contribution. Feel free to release a new version.

**Comment by oleg-nenashev** [comment]
The new version has been released.
BTW, the bug seems to be quite painful, because the code in envinject-lib is not protected from the null list. Currently I'm annotating the code in order to find the real cause of the issue

**Comment by dshvedchenko** [comment]
thank you for releasing it

**Comment by oleg-nenashev** [comment]
@dshvedchenko 
Could you list all Project type plugins you use?
I suppose the issue happens for a plugin, which overrides <code>AbstractBuild::getSensitiveBuildVariables()</code> and returns <code>null</code>. The core's Javadoc does not describe such situation

**Comment by dshvedchenko** [comment]
those are option when for New Item: hope it will help
* Freestyle project
* MultiJob Project
* Maven project
* External Job
* Folder
* Multi-configuration project

**Comment by oleg-nenashev** [comment]
OK, thanks a lot. I'll submit patches to both Jenkins core and envinject-lib

**Pull request merged**